### ISSUE TITLE
Build / install instructions?

### ISSUE THREAD

**Issue by ewa** [proj3rd/tool3rd#33]
Hi,

Do you have instructions for how to run this?  The videos only cover using it once it's running.  I tried the logical steps of checking out the repository and running `yarn run start` (and `yarn install` and `yarn build`) in the top-level directory (using both `main` and `v1.17.1`) but ran into a variety of peer dependency errors doing so.

Thanks!

**Comment by sjeon87** [comment]
Hello, thanks for your attention to this project.

You can just download the installer [here](https://github.com/proj3rd/tool3rd/releases).

Or, do you want to build binary for Linux or macOS?

**Comment by ewa** [comment]
You bet!

macOS in my case (but I imagine Linux and Windows users would also want to do it from time to time.)

**Comment by sjeon87** [comment]
I tried to build Linux and macOS executables, but found that lots of dependencies have breaking changes and it is very hard to fix dependencies.

I am going to rework the entire codebase and let you know if there is any progress

**Comment by ewa** [comment]
Awesome / good luck!  Don't feel like you need to do that on my account ... but I'll definitely use it if you do! 😁

**Comment by sjeon87** [comment]
Hello, try the followings (I don't have mac machine so I can't build for it):

## Clone & install dependencies

```sh
git clone https://github.com/proj3rd/tool3rd
cd tool3rd
git checkout develop
npm install
```

## Run on dev

```
npm run dev
```

## Build executables

```
npm build:win
npm build:linux
npm build:mac
```

If you encounter any issue, please let me know

Also, you can get Windows or Linux executables on https://github.com/proj3rd/tool3rd/releases/tag/v2.0.0-alpha

**Comment by sjeon87** [comment]
If available, could you check #34 on mac?

**Comment by sjeon87** [comment]
@ewa , could you check https://github.com/proj3rd/tool3rd/releases/tag/v2.1.0 ?

I built macOS app using GitHub workflow with macOS runner, but it's impossible to test whether it works or not.
